src/main/java/fsm/util/PropertiesUtil.java

```diff
@@ -0,0 +1,18 @@
+package fsm.util;
```

Comment:
Read these questions to correctly implement the property file implementation in spring. This will be helpful to configure hibernate correctly.
http://stackoverflow.com/questions/13297157/best-ways-to-deal-with-properties-values-in-xml-file-in-spring-maven-and-eclips
http://stackoverflow.com/questions/9259819/how-to-read-values-from-properties-file